nilearn/datasets.py

```diff
@@ -1066,6 +1066,38 @@ def fetch_smith_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
+def fetch_atlas_power_2011(verbose=1):
+    """Download and load the Power et al. brain atlas composed of 264 ROIs.
+
+    Parameters
```

Comment:
The docstring is wrong: no data_dir or url parameter.

Also the 'verbose' parameter is not used.